src/wwElement.vue

```diff
@@ -260,6 +261,7 @@ export default {
         handleCellClick(event) {
             const date = 'date' in event ? event.date : event;
             const calendar = 'split' in event ? event.split : null;
+            this.internalSelectedDate = event;

```

Comment:
are we sure this is the only way for the value to be modify? For example changing month does not update current date?

Reply:
@alexyca You need to check if the @cell-click handle as many case as the v-model handled, for exemple if you click on the Today button or next/previous does it still change the internalSelectedDate and so the component variable "selectedDate"

Reply:
@AlexisJC @AurelieV 

I updated the fix and put again v-model but added a debounce as it seems to be the only solution to avoid the black-boxed behavior of vue cal that triggered two selectedDate updates

so we keep all sync, just prevent the month skip that we had